README.md

```diff
@@ -7,6 +7,10 @@
 - about my self
 - contact me
 
+# Deploy Version
+- see the project [portfolio](https://girma336.github.io/)
+- deploy link [deploy](https://github.com/girma336/girma336.github.io/actions/runs/3219817472)
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure the link in blue is the one you put in your README file and not the one redirecting the user to the action page. You can see the link in the screenshot :point_down: 
![image](https://user-images.githubusercontent.com/97100091/194886552-d9e393bd-589a-4cb5-86dd-0f25d9665ecc.png)